MangoAPI.Client/src/app/components/chats/chats.component.ts

```diff
@@ -545,6 +588,22 @@ export class ChatsComponent implements OnInit {
     }
   }
 
+  onTypingHandler(event: KeyboardEvent) {
```

Comment:
Bind this handler to message text input value changed, see example https://github.com/MangoInstantMessenger/MangoMessengerAPI/blob/develop/MangoAPI.Client/src/app/components/contacts/contacts.component.html#L38